test/dlc_tests/container_tests/bin/benchmark/run_pytorch_training_performance_gpu_imagenet

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+PYTHON_VERSION=$(python -c 'import sys; print(sys.version_info[0])' | tr -d "'")
+if [ "$PYTHON_VERSION" -eq 2 ]
+then
+  exit 0
+fi
+TIMESTAMP=$(date "+%Y-%m-%d-%H-%M-%S")
+HOME_DIR=/test/benchmark
+BIN_DIR=${HOME_DIR}/bin
+LOG_DIR=${HOME_DIR}/logs/gpu
+LOG_FILE=image_results_${COMMIT_INFO}_${TIMESTAMP}.txt
+
+mkdir -p ${HOME_DIR}
+mkdir -p ${BIN_DIR}
+mkdir -p ${LOG_DIR}
+
+cd ~
+echo Benchmark Preparation started: >&2
```

Comment:
To run the script twice and exit forcely for the 1st round is to avoid the stuck of the testing script, which is likely to happen on a new launched AMI. After that, 2nd 2round would run smoothly

Reply:
Can we avoid duplication here by parameterizing the timeout?